Extensions/TileMap/JsExtension.js

```diff
@@ -1540,34 +1540,38 @@ module.exports = {
     // GDJS doesn't use Promise to avoid allocation.
     RenderedCollisionMaskInstance.prototype._loadTiledMapWithCallback =
       function (tilemapJsonFile, tilesetJsonFile, callback) {
-        this._loadTiledMap(tilemapJsonFile, tilesetJsonFile).then(callback);
+        this._loadTileMap(tilemapJsonFile, tilesetJsonFile).then(callback);
       };
 
-    RenderedCollisionMaskInstance.prototype._loadTiledMap = async function (
+    RenderedCollisionMaskInstance.prototype._loadTileMap = async function (
       tilemapJsonFile,
       tilesetJsonFile
     ) {
-      let tileMapJsonData = null;
       try {
-        tileMapJsonData = await this._pixiResourcesLoader.getResourceJsonData(
+        const tileMapJsonData = await this._pixiResourcesLoader.getResourceJsonData(
           this._project,
           tilemapJsonFile
         );
 
-        const tilesetJsonData = tilesetJsonFile
-          ? await this._pixiResourcesLoader.getResourceJsonData(
-              this._project,
-              tilesetJsonFile
-            )
-          : null;
+        const tileMap = TilemapHelper.TileMapManager.identify(tileMapJsonData);
```

Comment:
This is the fix, the other changes is just to clean the code a bit.

This method was not adapted when the LDTK handling was added and the Tiled data was returned instead of a tuple.